sdb/commands/count.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Count(sdb.Command):
+    # pylint: disable=too-few-public-methods
+
+    names = ["count", "cnt", "wc"]
+
+    def call(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
+        count = 0
+        for _ in objs:
+            count += 1
```

Comment:
would it be better to use `sum` instead? e.g.
```
sum(1 for _ in objs)
```
I don't know if it matters much, but it's one line vs. three...

Reply:
Nice! I applied this change